backon/src/sleep.rs

```diff
@@ -36,7 +36,11 @@ impl<F: Fn(Duration) -> Fut + 'static, Fut: Future<Output = ()>> Sleeper for F {
 /// The default implementation of `Sleeper` when no features are enabled.
 ///
 /// It will fail to compile if a containing [`Retry`][crate::Retry] is `.await`ed without calling [`Retry::sleep`][crate::Retry::sleep] to provide a valid sleeper.
-#[cfg(all(not(feature = "tokio-sleep"), not(feature = "gloo-timers-sleep"),))]
+#[cfg(all(
+    not(feature = "tokio-sleep"),
+    not(feature = "gloo-timers-sleep"),
+    not(feature = "futures-timer-sleep")
```

Comment:
We don't need to change this. `futures-timer-sleep` shoule not be part of dafault timer.

Reply:
Done.